src/deepcell/cli/modules/create_train_test_split.py

```diff
@@ -0,0 +1,215 @@
+    )
+
+    @marshmallow.post_load
+    def get_model_inputs(self, data):
+        """Load the json blob labeling data from the app database and return
+        the data as ModelInput objects.
+        """
+        model_inputs = []
+        # Get all the potential regions.
+        db_engine = db.create_engine(f"sqlite:///{data['label_db']}")
+        region_query = db.select([JobRegion.id, JobRegion.experiment_id])
+        region_data = db_engine.execute(region_query).fetchall()
+
+        for (region_id, exp_id) in region_data:
+            # For each region, load the user labels from the database.
+            label_query = db.select([UserLabels.user_id,
+                                     UserLabels.region_id,
+                                     UserLabels.labels])
+            label_query = label_query.where(
+                UserLabels.region_id == region_id)
+            label_results = db_engine.execute(label_query).fetchall()
+            # Count the number of times this region was classified by a
+            # User.
+            n_labelers = len(label_results)
+            if n_labelers < data["min_labelers_required_per_region"]:
+                continue
+            # Load the labeling data if the number of classifiers is
+            # sufficient.
+            region_labels = [pd.DataFrame(
+                                 data=json.loads(labels)).set_index('roi_id')
+                             for _, _, labels in label_results]
+            for roi_id, roi_row in region_labels[0].iterrows():
+                # Don't create a ModelInput if the label is a False Negative.
+                if not roi_row['is_segmented']:
+                    continue
+                # Count the number of votes for "cell". If the percentage
+                # of votes is sufficient, set the ROI label to cell.
+                n_votes = 0
+                label = self._tally_votes(
+                    roi_id=roi_id,
+                    region_labels=region_labels,
+                    n_labelers=n_labelers,
+                    min_labelers_per_region=\
+                        data['min_labelers_required_per_region'],
+                    vote_threshold=data['vote_threshold'])
+
+                # Create the output model input for the ROI.
+                model_input = ModelInput.from_data_dir(
+                    data_dir=data['artifact_dir'],
+                    experiment_id=str(exp_id),
+                    roi_id=str(roi_id),
+                    label=label)
+                model_inputs.append(model_input)
+        data['model_inputs'] = model_inputs
+        return data
+
+    @staticmethod
+    def _tally_votes(roi_id: int,
+                     region_labels: List[pd.DataFrame],
+                     n_labelers: int,
+                     min_labelers_per_region: int,
+                     vote_threshold: int):
+        """Loop through labels for a given roi_id and return if they were
```

Comment:
That's not actually what this method is doing; it is returning a final label given multiple labeler's votes for a given roi